app/__init__.py

```diff
@@ -14,6 +15,18 @@
 login_manager = LoginManager()
 login_manager.session_protection = 'strong'
 login_manager.login_view = 'auth.login'
+oauth = OAuth()
+github = oauth.remote_app(
+        'github',
+        consumer_key='e4dab2c621b44a75bd51',
+        consumer_secret='bde0d04cd12e0c8e887eba324af849356dd16d10',
```

Comment:
These should be environment variables, you should never expose keys and secrets like this. See this for an example https://github.com/levlaz/braindump/blob/master/config.py#L10.